src/api/ipfs.actions.ts

```diff
@@ -0,0 +1,23 @@
+import { create as createIpfsClient } from 'ipfs-http-client'
```

Comment:
I ran into some challenges using `js-ipfs` / `ipfs-core`, which I assume we'll want to use when writing, but this seems to work fine